### PR TITLE
fix: update code generation for Java and WDIO frameworks

### PR DESCRIPTION
**Pull request by zeufack**
Update java code generator framework 

-change Mobile Element to Web Element

- Change Touch Action to use Sequence

### CODE REVIEW

File: app/renderer/lib/client-frameworks/java.js [hunk 1]
```diff
@@ -107,20 +109,51 @@ ${this.indent(code, 4)}
...
+    (new TouchAction(driver)).tap(${x}, ${y}).perform()
+    Point tapPoint = new Point(${x}, ${y});
+    Sequence sequence = new Sequence(PointerInput.Kind.TOUCH, 1);
+    sequence.addAction(finger.createPointerMove(Duration.ofMillis(0),
+          PointerInput.Origin.viewport(), tapPoint.x, tapPoint.y));
+    sequence.addAction(finger.createPointerDown(PointerInput.MouseButton.LEFT.asArg()));
+    sequence.addAction(finger.createPointerMove(Duration.ofMillis(50),
+          PointerInput.Origin.viewport(), tapPoint.x, tapPoint.y));
+    sequence.addAction(finger.createPointerUp(PointerInput.MouseButton.LEFT.asArg()));
+    driver.perform(Arrays.asList(swipe))
```
Comment (jlipps) [inline app/renderer/lib/client-frameworks/java.js:126]: should `swipe` be `sequence` here?

File: app/renderer/lib/client-frameworks/java.js [hunk 2]
```diff
@@ -107,20 +109,51 @@ ${this.indent(code, 4)}
...
 
-    return `(new TouchAction(driver))
-  .press(PointOption.point(${x1}, ${y1}}))
-  .moveTo(PointOption.point(${x2}, ${y2}}))
-  .release()
-  .perform();
+    return `
+    Point start = new Point(${x1}, ${y1});
+    Point end = new Point (${x2}, ${y2});
+    Sequence swipe = New  Sequence(FINGER, 1)
```
Comment (jlipps) [inline app/renderer/lib/client-frameworks/java.js:137]: `New` should not be capitalized

File: target/npmlist.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"version":"2023.7.1","name":"appium-inspector","dependencies":{"@reduxjs/toolkit":{"version":"1.9.5"},"@xmldom/xmldom":{"version":"0.8.10"},"antd":{"version":"4.24.12"},"bluebird":{"version":"3.7.2"},"cheerio":{"version":"1.0.0-rc.10"},"electron-log":{"version":"4.4.8"},"electron-settings":{"version":"4.0.2"},"electron-updater":{"version":"6.1.1"},"format-json":{"version":"1.0.3"},"highlight.js":{"version":"11.8.0"},"htmlparser2":{"version":"7.2.0"},"i18next-chained-backend":{"version":"4.4.0"},"i18next-fs-backend":{"version":"2.1.5"},"i18next-http-backend":{"version":"2.2.1"},"i18next-localstorage-backend":{"version":"4.1.1"},"i18next":{"version":"23.2.11"},"ky":{"version":"0.25.1"},"lodash":{"version":"4.17.21"},"moment":{"version":"2.29.4"},"react-dom":{"version":"18.2.0"},"react-i18next":{"version":"13.0.2"},"react-icons":{"version":"4.10.1"},"react-redux":{"version":"8.1.1"},"react-router-dom":{"version":"6.14.2"},"react":{"version":"18.2.0"},"request-promise":{"version":"0.0.1"},"request":{"version":"2.88.2"},"semver":{"version":"7.5.4"},"uuid":{"version":"9.0.0"},"web2driver":{"version":"3.0.4"},"xpath":{"version":"0.0.33"}}}
```
Comment (jlipps) [inline target/npmlist.json:1]: this file should not be committed

**Review by jlipps — CHANGES_REQUESTED** [review]
thank you! I appreciate these changes! I think I found a couple errors. Can you actually get this generated code and then run it in Java / JS to make sure it compiles and runs correctly?

File: app/renderer/lib/client-frameworks/java.js [hunk 4]
```diff
@@ -39,9 +40,18 @@ public class SampleTest {
     DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
 ${capStr}
 
-    URL remoteUrl = new URL("${this.serverUrl}");
+    
 
-    driver = new ${cls}(remoteUrl, desiredCapabilities);
+    private URL remoteUrl() {
+      try {
+          return new URL("http://127.0.0.1:4723");
```
Comment (jlipps) [inline app/renderer/lib/client-frameworks/java.js:47]: the server and port information should not be hardcoded

File: target/npmlist.json (hunk 3 above)
Comment (jlipps) [reply target/npmlist.json:1]: please remove this file from your changeset

**Review by jlipps — CHANGES_REQUESTED** [review]

**Review by eglitise — CHANGES_REQUESTED** [review]
The file `target/npmlist.json` still exists and should be removed

**Comment by eglitise** [comment]
Simply delete the `target` folder in your local workspace and commit that action.

**Comment by zeufack** [comment]
> Simply delete the `target` folder in your local workspace and commit that action.

**Review by jlipps — APPROVED** [review]

**Comment by jlipps** [comment]
thanks @zeufack !

**Comment by zeufack** [comment]
oh, it's me who thanks you
I can't wait to work on other subjects

**Pull request merged**